src/createServer.js

```diff
@@ -1,3 +1,63 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+/* eslint-disable max-len */
+const http = require('http');
+const { convertToCase } = require('./convertToCase/convertToCase');
+
+const createServer = () => {
+  const server = http.createServer((req, res) => {
+    res.setHeader('Content-Type', 'application/json');
+
+    const normalizedURL = new URL(req.url, `http://${req.headers.host}`);
+
+    const targetCase = normalizedURL.searchParams.get('toCase');
+    const originalText = normalizedURL.pathname.slice(1);
+
+    const error = {
+      errors: [],
+    };
+
+    const casesToConvert = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+    if (!originalText.length) {
+      const newError = {
+        message: 'Text to convert is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+      };
+
+      error.errors.push(newError);
+    }
+
+    if (targetCase === null) {
```

Comment:
```suggestion
    if (!targetCase) {
```

Reply:
IDK why, but it didn`t work and tests failed with this condition(

Reply:
:smiling_face_with_tear:

Reply:
I fixed some moments and now everything is OK)